byron/ledger/executable-spec/src/Ledger/Update.hs

```diff
@@ -673,6 +721,37 @@ instance STS UPIVOTE where
 instance Embed UPVOTE UPIVOTE where
   wrapFailed = UPVOTEFailure
 
+
+data UPIVOTES
+
+instance STS UPIVOTES where
+  type Environment UPIVOTES = UPIEnv
+  type State UPIVOTES = UPIState
+  type Signal UPIVOTES = [Vote]
+
+  data PredicateFailure UPIVOTES
+    = UpivoteFailure (PredicateFailure UPIVOTE)
+    deriving (Eq, Show)
+
+  -- There is a rule in the LaTeX document for this STS that makes no
+  -- change to the state without any preconditions, hence this rule is
+  -- not imeplemtened as it is a no-op.
```

Comment:
OK, looking in more detail, it seems that what you actually mean is that you've folded the two rules into one. As it is, this comment is deeply misleading, and somewhat harmful since no-op rules _must_ be implemented to preserve the system being able to move forward.

Reply:
You're right and this is an outdated comment, as you noticed by comparing it with the implementation. I'm removing the comment...